defaults/main.yml

```diff
@@ -465,7 +465,14 @@ apache__combined_snippets: '{{ apache__dependent_snippets
 apache__https_enabled: '{{ ansible_local|d() and ansible_local.pki|d() and
                            (ansible_local.pki.enabled|d() | bool) and
                            apache__https_listen|length > 0 }}'
+                                                                   # ]]]
+
```

Comment:
Seems to be that `yaml4rst` is run by Python2 which is unsupported. Please run it with Python3.